pom.xml

```diff
@@ -600,6 +608,7 @@
             <id>mysql</id>
             <properties>
                 <database.container>mysql</database.container>
+                <osiam.test.host>${osiam.host.protocol}://${osiam.host}</osiam.test.host>
```

Comment:
Absolutely right, I removed both and added the property to the `properties` element